.github/workflows/PR_build_extension.yml

```diff
@@ -77,6 +76,38 @@ jobs:
           artifacts: |
             ./packages/vscode/prisma.vsix
 
+      # Additionally post a comment with a link to GitPod that can use built extension instantly
+      - name: Create comment body
+        if: "!contains(github.actor, 'renovate')"
+        id: comment
+        run: |
+          body="[Try this extension build via GitPod](https://gitpod.io/#vsix=https%3A%2F%2Fgithub.com%2Fprisma%2Flanguage-tools%2Fblob%2Fartifacts%2Fpull-request-artifacts%2Fpr${{ github.event.pull_request.number }}-prisma.vsix%3Fraw%3Dtrue/https://github.com/prisma/language-tools-gitpod)"
+          echo "::set-output name=body::$body"
+        
+      - name: Find existing comment
+        if: "!contains(github.actor, 'renovate')"
+        uses: peter-evans/find-comment@v1
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: "via GitPod"
+
+      - name: Create comment
+        if: "!contains(github.actor, 'renovate') && steps.fc.outputs.comment-id == ''"
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: ${{ steps.comment.outputs.body }}
+
+      - name: Update comment
+        if: "!contains(github.actor, 'renovate') && steps.fc.outputs.comment-id != ''"
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          body: ${{ steps.comment.outputs.body }}
+          edit-mode: replace
+      
```

Comment:
@janpio It's a bit more complicated than I thought, why not only creating once? Since the link doesn't need to be updated?

Reply:
Good point.